python/numpy-in-pytorch-modules.yaml

```diff
@@ -15,7 +15,9 @@ rules:
         - https://tanelp.github.io/posts/a-bug-that-plagues-thousands-of-open-source-ml-projects
 
     patterns:
-      - pattern: $RESULT = numpy.$FUNCTION(...)
+      - pattern-either:
+          - pattern: numpy.$FN(...)
+          - pattern: numpy.$MOD.$FN(...)
```

Comment:
You can also use something like `numpy. ... .$FN(...)` if you'd like to match arbitrary depth.

Reply:
Good point, fixed